notebooks/external_innervation.ipynb

```diff
@@ -235,11 +302,12 @@
     "layers = ['SO', 'SP', 'SR', 'SLM']\n",
     "mod_y_distr = np.array([np.count_nonzero(synapses.layer==layer) for layer in layers], dtype=np.float32)\n",
     "mod_y_distr = mod_y_distr/mod_y_distr.sum()\n",
-    "exp_y_distr = [0.2476014415, 0.0705293209, 0.6792951748, 0.0025740628]\n",
+    "exp_y_distr = [0.2476014415, 0.0705293209, 0.6792951748, 0.0025740628] # Bezaire and Soltesz, 2013\n",
     "x = np.arange(4)\n",
+    "width = 0.35\n",
     "fig, ax = plt.subplots()\n",
-    "ax.bar(x, mod_y_distr, label='simulation')\n",
-    "ax.plot(x,exp_y_distr,color='r', label='expected')\n",
+    "ax.bar(x-width/2, mod_y_distr, width, label='simulation')\n",
+    "ax.bar(x+width/2, exp_y_distr, width, color='r', label='expected')\n",
```

Comment:
Maybe the plot is a bit more clear when the y axis is log-scale? 
ax.semilogy()

Reply:
I would prefer to keep linear axis. I prefer to stress the fact that most of the synapses are in SR and SO.

Reply:
Sure, makes sense